Tests/SSL/test_data_modules.py

```diff
@@ -32,8 +32,8 @@ def test_weights_innereye_module() -> None:
     """
     Tests if weights in CXR data module are correctly initialized
     """
-    transforms = get_cxr_ssl_transforms(cxr_augmentation_config,
-                                        return_two_views_per_sample=True)
+    transforms = get_ssl_transforms_from_config(cxr_augmentation_config,
```

Comment:
augmentation_config? Why cxr?

Reply:
That config has been defined for cxr, here I am importing it from Tests.SSL.test_data_modules and reusing it. It's used in other tests, that are specific to cxr, I think renaming it to something more generic could be confusing